src/test/resources/network/brightspots/rcv/test_data/2013_minneapolis_mayor_scale/2013_minneapolis_mayor_scale_expected_summary.csv

```diff
@@ -1,63 +1,63 @@
+CYD GORMAN,507,0.04%,0,507,0.04%,0,507,0.04%,-507,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0
+JOHN CHARLES WILSON,481,0.04%,13,494,0.04%,-494,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0
+Undeclared Write-ins,1521,0.14%,-1521,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0,0,0.0%,0
+Active Ballots,1031056,,,1030575,,,1030497,,,1030432,,,1030289,,,1030159,,,1029938,,,1029782,,,1029561,,,1029132,,,1028755,,,1028326,,,1027845,,,1027559,,,1027039,,,1026376,,,1025999,,,1025674,,,1024738,,,1024036,,,1023308,,,1022021,,,1020409,,,1016899,,,1015053,,,1013480,,,1006863,,,1002053,,,996099,,,992199,,,982670,,,957047,,,829322,,
+Current Round Threshold,515529,,,515288,,,515249,,,515217,,,515145,,,515080,,,514970,,,514892,,,514781,,,514567,,,514378,,,514164,,,513923,,,513780,,,513520,,,513189,,,513000,,,512838,,,512370,,,512019,,,511655,,,511011,,,510205,,,508450,,,507527,,,506741,,,503432,,,501027,,,498050,,,496100,,,491336,,,478524,,,414662,,
+Inactive Ballots by Overvotes,1339,,13,1352,,0,1352,,0,1352,,0,1352,,0,1352,,0,1352,,0,1352,,0,1352,,13,1365,,13,1378,,13,1391,,0,1391,,0,1391,,0,1391,,0,1391,,0,1391,,0,1391,,0,1391,,0,1391,,0,1391,,13,1404,,13,1417,,13,1430,,0,1430,,0,1430,,13,1443,,13,1456,,26,1482,,13,1495,,26,1521,,78,1599,,221,1820,,0
+Inactive Ballots by Skipped Rankings,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0,611,,0
+Inactive Fully Ranked Ballots by Exhausted Choices,0,,234,234,,0,234,,13,247,,13,260,,52,312,,65,377,,65,442,,26,468,,195,663,,182,845,,195,1040,,143,1183,,208,1391,,351,1742,,273,2015,,273,2288,,247,2535,,364,2899,,611,3510,,598,4108,,1157,5265,,1313,6578,,2093,8671,,1300,9971,,1235,11206,,3978,15184,,4368,19552,,3393,22945,,3068,26013,,6955,32968,,18278,51246,,101751,152997,,0
+Inactive Partially Ranked Ballots by Exhausted Choices,0,,234,234,,78,312,,52,364,,130,494,,78,572,,156,728,,91,819,,195,1014,,221,1235,,182,1417,,221,1638,,338,1976,,78,2054,,169,2223,,390,2613,,104,2717,,78,2795,,572,3367,,91,3458,,130,3588,,117,3705,,286,3991,,1404,5395,,546,5941,,338,6279,,2626,8905,,429,9334,,2535,11869,,819,12688,,2548,15236,,7267,22503,,25753,48256,,0
```

Comment:
This concerns me -- it must be a fluke of the construction of these files, so I should investigate. Why is the Fully Ranked Exhausted exactly equal Partially Ranked Exhausted?